wearos_app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
This should be added to the main `.gitignore` file rather than creating a new one.

Reply:
I just checked. It is currently in the main .gitignore as
`build/`
But it's also in app/.gitignore, buildSrc/.gitignore and common/.gitignore as
`/build`

Do you want me to remove it in all subdirs?

Reply:
Yeah, I didn't even notice there was one in `app/`.  If you could remove all the sub dir ones that would be great, ty!